### DISCUSSION TITLE
Toggle Switch for active inactive

### DISCUSSION THREAD

**Discussion by AhmedMub** [rappasoft/laravel-livewire-tables#800] [Q&A]
Hi, I really wanted to thank you for such amazing work in datatable.

Is there a way of adding toggle Switch for active and inactive for boolean columns instead of displaying icons?!

![image](https://user-images.githubusercontent.com/32661307/172117672-062353a3-c6b7-4be5-83f1-5a34f52db338.png)

**Comment by rappasoft** [comment]
You can return a custom livewire component to the cell to do it and then just trigger a table refresh.
